### DISCUSSION TITLE
Allow configuring supported connectors

### DISCUSSION THREAD

**Discussion by dirtycajunrice** [family/connectkit#20] [Ideas]
There is no way to add supported connectors. If i wanted to add gnosis safe, i should be able to append to the list of supported connectors with a type safe supported connector definition

**Comment by lochie** [comment]
Hey @dirtycajunrice,

We are looking towards setting ConnectKit up to allow for custom wallet connectors, but there's just a little bit of refactoring needed in our architecture to have this be setup as a friendly developer option.

If you have a wallet you want added to ConnectKit's default setup, we have a directory with [supported wallets configs](https://github.com/family/connectkit/tree/main/packages/connectkit/src/wallets/connectors) (includes [Gnosis Safe](https://github.com/family/connectkit/blob/main/packages/connectkit/src/wallets/connectors/gnosisSafe.tsx)!) and we are happy to review PRs for wallets that want to be shipped with ConnectKit. 😊

Thanks!
